docs/contributing.mdx

```diff
@@ -126,11 +126,7 @@ MDX is a monorepo that uses [lerna][].
 In order to release a new version you can follow these steps:
 
 - Draft a release for the next version (vX.X.X)
-- Release a prerelease
-  - `yarn lerna publish`
-  - Select prepatch/preminor/premajor
-  - Sanity check in a project or two with the prerelease
-- `yarn lerna publish`
+- `yarn && yarn test && yarn yarn run publish`
```

Comment:
@laurieontech `yarn yarn (run publish)`? Is that a thing 😅

Reply:
Ha! We both missed it. Will fix again.